src/domUpdates.js

```diff
@@ -35,16 +40,60 @@ let domUpdates = {
       <section class='booked-info-card'>
         <p>Booking For ${booking.date}</p>
         <p>Confirmation ID: ${booking.id}</p>
-        <p>Room Type ${foundRoom.roomType}</p>
-        <p>Cost ${foundRoom.costPerNight}</p>
+        <p>Room Type: ${foundRoom.roomType}</p>
+        <p>Cost: $ ${foundRoom.costPerNight}</p>
       </section>
       `
     })
+  },
+
+  displayAvailableRooms(currentCustomer, roomsData, bookingsData) {
+    availableRoomsSection.innerHTML = '';
+    const date = selectedDate.value.split('-').join('/');
+    currentCustomer.getAvailableRooms(date, roomsData, bookingsData);
+    console.log('customer', currentCustomer)
+
+    console.log('customer', currentCustomer.availableRooms)
+    currentCustomer.availableRooms.forEach((room) => {
+      availableRoomsSection.innerHTML += `
+      <section class='individual-room-cards'>
+        <p>${room.roomType}</p>
+        <p>has a bidet: ${room.bidet}</p>
+        <p>${room.bedSize} size bed</p>
+        <p>number of beds: ${room.numBeds}</p>
+        <p>cost per night: ${room.costPerNight}</p>
+      </section>
```

Comment:
I think that this is actually displaying different information than lines 41-59. Only two lines are the same, roomType and costPerNight, and even those are displayed differently.  I think you would be better off leaving them as separate functions.

But if you really want to make it one, I keep coming back to an if-statement, based on page load or room type selection button click, utilizing += and -= to change innerHTML.

Reply:
Both fantastic points. Since only those two lines are the same, I feel like it's actually likely more code (or at least lines of code?) to put it all in a conditional